changelogs/readme.md

```diff
@@ -1,16 +1,23 @@
 # Update changelog
-After each release, call:
+After releasing x1.y1.z1, assuming next release will be x2.y2.z2:
 ```
-git mv changelog.md changelogs/changelog_0_19_0.md # use correct version
-cp changelogs/changelog_X_XX_X.md changelog.md
-git add changelog.md
+cp changelogs/changelog_X_XX_X.md changelogs/changelog_x2_y2_z2.md
+update 1st line in changelog.md to point to changelogs/changelog_x2_y2_z2.md
+update system.NimMinor,NimMinor,NimPatch if needed
 ```
 
 ## Recent changelogs are saved here (with their git history)
 
-## Older changelogs are stored in https://github.com/nim-lang/website
+## Released changelogs
+released changlogs are copied to the nim website repo, see list in
```

Comment:
Sentences start with the capital case. C'mon.

Reply:
done